pkg/factories/flags_factory.go

```diff
@@ -48,6 +48,8 @@ func (f *kafkaSourceFlagsFactory) CreateFlags() *pflag.FlagSet {
 	flagSet.StringArrayVar(&f.KafkaSourceParams().BootstrapServers, "servers", []string{}, "Kafka bootstrap servers that the consumer will connect to, consist of a hostname plus a port pair, e.g. my-kafka-bootstrap.kafka:9092. Flag can be used multiple times.")
 	flagSet.StringArrayVar(&f.KafkaSourceParams().Topics, "topics", []string{}, "Topics to consume messages from. Flag can be used multiple times.")
 	flagSet.StringVar(&f.KafkaSourceParams().ConsumerGroup, "consumergroup", "", "the consumer group ID")
+	flagSet.StringArrayVar(&f.KafkaSourceParams().CeOverrides, "ce-override", []string{}, "Cloud Event Overrides defines overrides to control the output format \n"+
```

Comment:
Newline is breaking lint check here.

Reply:
Also we could align the description with other `ce-override` flags. https://github.com/knative/client/blob/main/pkg/kn/commands/source/binding/flags.go#L42-L45

And finally it might be worth extract the flag in `client` to make it re-usable.(not necessarily in scope of this PR)